tutorials/guacamole-tutorial/pom.xml

```diff
@@ -62,15 +62,15 @@
         <dependency>
             <groupId>org.apache.guacamole</groupId>
             <artifactId>guacamole-common</artifactId>
-            <version>1.1.0</version>
+            <version>1.2.0</version>
```

Comment:
Depending on apache/guacamole-client#475, this may need to remain 1.1.0. Other than that, these changes look good to me.